.github/workflows/sync-generated-files.yml

```diff
@@ -0,0 +1,38 @@
+name: Sync generated files
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          version: 10.x
+      - name: Install dependencies
+        run: yarn install --offline
```

Comment:
Looks good. I was just curious why in the circle ci config we had to do `--frozen-lockfile` but not here?

Reply:
They're essentially the same thing here, `--offline` means we only download from a known folder where-as `--frozen-lockfile` will fail if `package.json` specifies a version outside of what is in `yarn.lock`. Really we only need the former haha